doc/api/cli.md

```diff
@@ -2252,6 +2257,39 @@ This option may be specified multiple times to include multiple glob patterns.
 If both `--test-coverage-exclude` and `--test-coverage-include` are provided,
 files must meet **both** criteria to be included in the coverage report.
 
+### `--test-coverage-branches=threshold`
```

Comment:
It just occurred to me - it seems like these new flags can be used without ever properly enabling experimental coverage with the `--experimental-test-coverage` flag. Maybe we should wait on making that an alias until coverage moves out of experimental status.